editor/editor_inspector.cpp

```diff
@@ -2266,6 +2266,21 @@ void EditorInspector::_edit_set(const String &p_name, const Variant &p_value, bo
 		undo_redo->add_do_property(object, p_name, p_value);
 		undo_redo->add_undo_property(object, p_name, object->get(p_name));
 
+		// Get additional properties that would need to be saved if needed.
+		List<PropertyChange> property_changes;
+		object->get_undo_property_list(p_name, object->get(p_name), p_value, property_changes);
+		for (List<PropertyChange>::Element *E = property_changes.front(); E; E = E->next()) {
+			const PropertyChange &change = E->get();
+			undo_redo->add_undo_property(change.object, change.property, change.value);
+		}
+
+		property_changes.clear();
+		object->get_undo_property_list(p_name, p_value, object->get(p_name), property_changes);
```

Comment:
This does not make sense to me, why would you set values on do ? as far as I understand all you need to do is set them on undo, and set them _after_ the property is changed (which so far is correct).

Reply:
Yeah I was not sure either if there was situations where this would be useful. I can remove it.